js/templates/events-page.mustache

```diff
@@ -3,7 +3,7 @@
     <div class="row margin-bottom-50">
     {{#image}}
         <div class="col-sm-12">
-            <img src="{{ image }}">
+            <img class="img img-responsive" src="{{ image }}">
```

Comment:
Why adding .img? I don't think that class exist in our framework..

Reply:
I thought they were both needed. I've always been adding these two classes to images lol...

I'll start removing them.